game/script-admissions.rpy

```diff
@@ -10,19 +10,19 @@ init 0 python in admissions:
     ADMISSION_MAP = dict()
 
     # Admission types
-    TYPE_ANGRY = 0
-    TYPE_ANXIOUS = 1
-    TYPE_ASHAMED = 2
-    TYPE_BORED = 3
-    TYPE_CONFIDENT = 4
-    TYPE_EXCITED = 5
-    TYPE_HAPPY = 6
-    TYPE_HUNGRY = 7
-    TYPE_INSECURE = 8
-    TYPE_PROUD = 9
-    TYPE_SAD = 10
-    TYPE_SICK = 11
-    TYPE_TIRED = 12
+    ADMISSION_TYPE_ANGRY = 0
+    ADMISSION_TYPE_ANXIOUS = 1
+    ADMISSION_TYPE_ASHAMED = 2
+    ADMISSION_TYPE_BORED = 3
+    ADMISSION_TYPE_CONFIDENT = 4
+    ADMISSION_TYPE_EXCITED = 5
+    ADMISSION_TYPE_HAPPY = 6
+    ADMISSION_TYPE_HUNGRY = 7
+    ADMISSION_TYPE_INSECURE = 8
+    ADMISSION_TYPE_PROUD = 9
+    ADMISSION_TYPE_SAD = 10
+    ADMISSION_TYPE_SICK = 11
+    ADMISSION_TYPE_TIRED = 12
```

Comment:
Since these are already in the `admissions` store, I'd rather not dual prefix this since we'd be accessing `admissions.TYPE_XYZ` instead of `admissions.ADMISSION_TYPE_XYZ` which doesn't really add much to verbosity due to the prefix